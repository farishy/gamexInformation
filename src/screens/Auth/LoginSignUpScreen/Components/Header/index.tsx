import React, {useEffect} from 'react';
import { Image, StyleSheet, View } from 'react-native';

const Header = () =>{
    return(
        <View style={styles.container}>
            <Image
                style={styles.headerLogo}
                source={require('../../../../../assets/images/logo_header.png')}
            />
        </View>
    )
};

const styles = StyleSheet.create({
    container:{
        alignItems:'center',
        backgroundColor:'white',
        width:'100%',
        height:90,
        borderBottomLeftRadius:50,
        borderBottomRightRadius:50,
        shadowColor: '#398FCE',
        borderColor:'#FFFFFF',
        borderBottomColor: '#398FCE',
        borderWidth:3,
        shadowOffset: {
            width: 0,
            height: 7,
        },
        shadowOpacity: 0.43,
        shadowRadius: 9.51,
        elevation: 25,
    },
    headerLogo:{
        marginTop:-20,
    },
});

export default Header;