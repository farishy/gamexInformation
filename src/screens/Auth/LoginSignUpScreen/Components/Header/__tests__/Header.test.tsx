import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../../Header';

test('header render', () => {
    const header = renderer
      .create(<Header data-testID={'header-loginSignup'}/>)
      .toJSON();
    expect(header).toMatchSnapshot();
  });
