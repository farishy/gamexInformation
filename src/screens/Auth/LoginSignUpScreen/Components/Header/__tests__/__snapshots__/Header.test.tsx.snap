// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header render 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "borderBottomColor": "#398FCE",
      "borderBottomLeftRadius": 50,
      "borderBottomRightRadius": 50,
      "borderColor": "#FFFFFF",
      "borderWidth": 3,
      "elevation": 25,
      "height": 90,
      "shadowColor": "#398FCE",
      "shadowOffset": Object {
        "height": 7,
        "width": 0,
      },
      "shadowOpacity": 0.43,
      "shadowRadius": 9.51,
      "width": "100%",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/images/logo_header.png",
      }
    }
    style={
      Object {
        "marginTop": -20,
      }
    }
  />
</View>
`;
